core/src/main/java/org/apache/calcite/rex/RexShuttle.java

```diff
@@ -225,6 +225,10 @@ public RexWindow visitWindow(RexWindow window) {
     return dynamicParam;
   }
 
+  @Override public RexNode visitNamedParam(RexNamedParam namedParam) {
```

Comment:
This should be correct. I don't understand the purpose of this visitor. It seems like it doesn't do anything unless extended?

Reply:
From what I can tell this just seems to update RexNodes if accept modifies the node. This shouldn't happen for RexNamedParam so it should be correct.